widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -137,4 +141,23 @@ public static void clearVisitorSession() {
         Dependencies.getControllerFactory().destroyControllers();
         Glia.clearVisitorSession();
     }
+
+    private static final Consumer<GliaException> updateVisitorExceptionConsumer = exception -> {
+        if (exception != null) {
+            Logger.e(TAG, "Visitor Info update failed: " + exception.toString());
+        } else {
+            Logger.e(TAG, "Visitor Info updated Success");
+        }
+    };
+
+    /**
+     * Updates the visitor's information
+     *
+     * Updates the visitor's information stored on the server. This information will also be displayed to the operator.
+     * */
+    public static void updateVisitorInfo(VisitorInfoUpdateRequest visitorInfoUpdateRequest) {
+        Logger.d(TAG, "updateVisitorInfo");
```

Comment:
This log message looks not so descriptive, at least for me.
Perhaps we could improve it in sense of adding some additional not sensitive information or remove it if this log doesn't provide any useful debug information.
Does it make sense?